.wordlist.txt

```diff
@@ -10,20 +10,26 @@ EBS
 EKS
 favicon
 Furo
+Git
+GitHub
 Grafana
+hostname
 IAM
 installable
 JSON
 Juju
 Kubernetes
 Kubeflow
+LaunchPad
```

Comment:
I've never seen Launchpad with the P capitalised before. I think it should be just Launchpad as used on the https://launchpad.net/ website?

Reply:
corrected